### ISSUE TITLE
node-sass is deprecated

### ISSUE THREAD

**Issue by kolodziejczak-sz** [fuse-box/fuse-box#2021]
Fuse-box enforces developer to install the deprecated package `node-sass` (more details here https://www.npmjs.com/package/node-sass)
 
 Having the recommended package `sass` (https://www.npmjs.com/package/sass) causes the following error:
 ```
  - Module "sass" is required, Please install it using the following command
  - npm install --save-dev node-sass
 ```

**Comment by insanity54** [comment]
I'm running into this problem as well. It looks like the source checks for `sass` first, and if that doesn't exist, it falls back to `node-sass`.

https://github.com/fuse-box/fuse-box/blob/b3cf2458e28ff0a1dd86c2db77611a921769e589/src/plugins/core/plugin_sass.ts#L22

However, it looks like the package I got from NPM `fuse@4.0.1-next.5` does not include the check for `sass`.

            if (props.module.captured || !matcher.test(module.absPath)) {
                return;
            }
            if (!utils_1.isNodeModuleInstalled('node-sass')) {
                ctx.fatal(`Fatal error when capturing ${module.absPath}`, [
                    'Module "sass" is required, Please install it using the following command',
                    `${env_1.getPackageManagerData().installDevCmd} node-sass`,
                ]);
                return;
            }
```

Not sure what's going on with that. Is the typescript build system using an old commit?

**Comment by kolodziejczak-sz** [comment]
You are right, it seems like they fixed it already, but this fix is not released yet - the 4.0.0 version doesn't include these changes.
